tools/theme-preview.sh

```diff
@@ -2,30 +2,104 @@
 #shellcheck disable=SC2317
 set -u
 
-. ./liquidprompt --no-activate
-
-# If a theme was given, use the theme and source all other arguments.
-if [[ -z ${1-} || $1 == --help ]]; then
+usage() {
   printf '
-Usage: %s theme [sourced files...]
+Usage: %s theme [options...] [sourced files...]
 
 Print out example prompts based on a standard set of input conditions. Designed
 to showcase Liquid Prompt themes.
 
-Example usage: %s powerline themes/powerline/powerline.theme
-' "$0" "$0"
-  exit 2
+Options:
+  -h, --help                 Print this help text.
+  --reproducible             Set the terminal size to a static value. Load no
+                             config files. Overrides any --config-file or
+                             --user-config arguments.
+  --user-config              Load the user, or if not found, the system config
+                             file.
+  --template-file=<filename> Load <filename> as a default theme template.
+                             Implies "default" theme.
+  --config-file=<filename>   Load <filename> as an additional Liquid Prompt
+                             config file.
+
+Example usage:
+%s alternate_vcs themes/alternate_vcs/alternate_vcs.theme
+%s unfold themes/unfold/unfold.theme --reproducible
+%s unfold themes/unfold/unfold.theme --user-config --config-file contrib/presets/colors/256-colors-dark.conf
+%s default --template-file templates/minimal/minimal.ps1
+' "$0" "$0" "$0" "$0" "$0"
+}
```

Comment:
Shouldn't usage messages be output on stderr? I am always hesitating about that.